src/batou_ext/run.py

```diff
@@ -41,6 +41,8 @@ def configure(self):
 
     def verify(self):
         self.parent.assert_no_changes()
+        self.assert_file_is_current(f'{self.command_file.path}_stamp', requirements=[f'.{self.command_file.path}',])
```

Comment:
Hum. I think that's wrong/broken.
`'.{self.command_file.path}'` refers to the generated script but is prefixed with `.`. I'd assume you wanted to prefix the stamp file with `.`. I suggest binding the stamp file path to a variable in configure, e.g. `self.map(f'.{self.…}_stamp')`.

Reply:
Indeed. I was using .{self.command_file.path} at every place earlier and fixed. Oviously missed one place